swagger-ext/src/main/scala/org/scalatra/swagger/SwaggerCommand.scala

```diff
@@ -63,7 +63,12 @@ object SwaggerCommandSupport {
 
   private[this] def modelFromCommand[T <: Command](cmd: T, fields: List[Parameter]) = {
     val modelFields = fields map { f =>
-      f.name -> ModelProperty(f.`type`, f.position, required = f.required, allowableValues = f.allowableValues)
+      f.name -> ModelProperty(
+        `type` = f.`type`,
+        position = f.position,
+        required = f.required,
+        default = f.defaultValue
+      )
```

Comment:
does this change follow scalatra's formatting style?
whitespace changes are pretty significant when contributing to a 3rd party project